src/components/Notifications/defaultStyle.ts

```diff
@@ -7,41 +7,22 @@ export default css`
 	.notification-container-bottom-center,
 	.notification-container-bottom-left,
 	.notification-container-bottom-right,
-	.notification-container-center,
```

Comment:
Make sure you install the latest npm modules, as this might be happening because you have older version (or maybe it is a newer version, not sure, just try and see :))

Reply:
I actually had an old version, thanks for the info :)